### ISSUE TITLE
F1 2018 (737800)

### ISSUE THREAD

**Issue by bfayers** [ValveSoftware/Proton#1567]
# Compatibility Report
- Name of the game with compatibility issues: F1 2018 - Works fine.
- Steam AppID of the game: 737800

## System Information
- GPU: GTX 1050Ti
- Driver/LLVM version: nVidia 390.48
- Link to full system information report as [Gist](https://gist.github.com/bfayers/bd113928e84cf861d2df13a21e5718b4):
- Proton version: 3.7-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-737800.log](https://github.com/ValveSoftware/Proton/files/2412484/steam-737800.log)

## Symptoms 
~~Game Launches fine but once launching benchmark or starting any session it crashes after loading.~~

## Reproduction
~~Launch F1 2018 with proton and try to run a benchmark or play a GP~~

Resolved via quoted comment below

> @libcg Should I edit the original post in this thread?
> 
> Running the 396.54 driver made the game work properly. The performance isn't the same as windows (inspect the two attached benchmark xml (renamed to .txt for github) files, both done with exactly the same settings) however I suppose this is to be expected when running via proton. Hopefully the performance can be improved somehow so that it could be playable on linux under proton. (60+fps minimum otherwise playing with a wheel gets odd, though admittedly the benchmark numbers are usually lower than real life, so ill try playing with my wheel later).
> 
> [LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt](https://github.com/ValveSoftware/Proton/files/2412185/LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt)
> [WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt](https://github.com/ValveSoftware/Proton/files/2412186/WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt)

**Comment by libcg** [comment]
Your nvidia drivers are too old, please follow the instructions here: https://github.com/ValveSoftware/Proton/blob/proton_3.7/PREREQS.md#nvidia

**Comment by bfayers** [comment]
@libcg Thanks I'll try this now, just the default Nvidia drivers Ubuntu gave me didn't think to check for newer versions.

**Comment by bfayers** [comment]
@libcg Should I edit the original post in this thread? 

Running the 396.54 driver made the game work properly. The performance isn't the same as windows (inspect the two attached benchmark xml (renamed to .txt for github) files, both done with exactly the same settings) however I suppose this is to be expected when running via proton. Hopefully the performance can be improved somehow so that it could be playable on linux under proton. (60+fps minimum otherwise playing with a wheel gets odd, though admittedly the benchmark numbers are usually lower than real life, so ill try playing with my wheel later).

[LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt](https://github.com/ValveSoftware/Proton/files/2412185/LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt)
[WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt](https://github.com/ValveSoftware/Proton/files/2412186/WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt)

**Comment by libcg** [comment]
Great! You're welcome to edit your post.

**Comment by bfayers** [comment]
@libcg Cool, i've edited my post. Guess ill submit a whitelist request later.

**Comment by kisak-valve** [comment]
Hello @bfayers, no, we'll do that here.

**Comment by bfayers** [comment]
@kisak-valve Cool, thanks.

**Comment by jomarocas** [comment]
> @libcg Should I edit the original post in this thread?
> 
> Running the 396.54 driver made the game work properly. The performance isn't the same as windows (inspect the two attached benchmark xml (renamed to .txt for github) files, both done with exactly the same settings) however I suppose this is to be expected when running via proton. Hopefully the performance can be improved somehow so that it could be playable on linux under proton. (60+fps minimum otherwise playing with a wheel gets odd, though admittedly the benchmark numbers are usually lower than real life, so ill try playing with my wheel later).
> 
> [LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt](https://github.com/ValveSoftware/Proton/files/2412185/LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt)
> [WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt](https://github.com/ValveSoftware/Proton/files/2412186/WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt)

i see your cpu is different from linux to windows, in linux appear a pentium in windows is a core i5, the benchmark is obviosuly winner a windows system

**Comment by bfayers** [comment]
> > @libcg Should I edit the original post in this thread?
> > Running the 396.54 driver made the game work properly. The performance isn't the same as windows (inspect the two attached benchmark xml (renamed to .txt for github) files, both done with exactly the same settings) however I suppose this is to be expected when running via proton. Hopefully the performance can be improved somehow so that it could be playable on linux under proton. (60+fps minimum otherwise playing with a wheel gets odd, though admittedly the benchmark numbers are usually lower than real life, so ill try playing with my wheel later).
> > [LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt](https://github.com/ValveSoftware/Proton/files/2412185/LINUX_MatchedSettings_Benchmark_2018-09-24_at_18-19-44.xml.txt)
> > [WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt](https://github.com/ValveSoftware/Proton/files/2412186/WINDOWS_Benchmark_2018-08-27_at_20-12-57.xml.txt)
> 
> i see your cpu is different from linux to windows, in linux appear a pentium in windows is a core i5, the benchmark is obviosuly winner a windows system

It's the same system ( dual booted ) ; wine makes my CPU show as a pentium for some reason

**Comment by bogdans41472** [comment]
I can attest that the game ran fine for me as well, albeit there was no audio. Running on a AMD 1700, Gigabyte Gaming K5 mobo and Vega 64 - drivers 18.40.2.15. Medium settings got me 74 avg fps on 4K resolution while Ultra settings got me around 39 avg fps at the same resolution.

**Comment by ppedro1** [comment]
On Antergos (4.19.4-arch1-1-ARCH) and latest NVIDIA drivers, the game ran on about 50 FPS on lowest settings. Some hangs here and there, so not really playable in this state. Also configuring my Logitech G29 steering wheel caused the game to crash.

**Comment by bfayers** [comment]
@ppedro1 
Yep, the framerate is pretty low making it pretty unplayable. Also the G29 doesn't work too good on linux however it definitely didn't make my game crash, you have to put the G29 in ps3 mode for linux to work well

**Comment by ppedro1** [comment]
@bfayers 
checking with `evtest` the G29 works flawlessy, but you are right, even in native racing games with wheel support, the wheel doesn't perform well. In F1 2017 (Feral Interactive's port) the wheel does work like it should, though.

Now that you mention it, I have set my G29 on PS4 cause I am playing F1 2018 on windows with it, should give the proton F1 a try with the PS3 mode set. Cheers.

Also, I am not saying the framerate makes the game unplayable, its the screen freezes which cause you to jump around the track if in MP (which happen in Windows also but not as much as with proton).

**Comment by bfayers** [comment]
@ppedro1 yep the freezes are pretty horriblethe framerate being low makes the input odd but i think that might not be aframerate thing. who knows.

**Comment by gotzl** [comment]
I'm using nvidia drivers 415.18 with a GTX1060 and Proton 3.7-8. On high quality settings, the framerate is stable around/above 60fps, however I experience some stuttering, for example in the tunnel of Monaco.
Also, the FFB of the DFGT is not working properly. The effects like road, curbs etc are all there, but the steering doesn't have any resistance. Any hints ?

**Comment by sebastian4699** [comment]
I can confirm that the game is running BUT there is a big lack of performance.
I have a i5 6600 with 3.3 GHz (also recognized as a Pentium 4 with 2.4 GHz by the game in linux), and a RX 580. In linux, I have roundabout 60 to 70 percent of the windows performance, just like bfayers. 
In high serttings I have fluent 60 fps and more, but in windows I can use ultra settings for that. With high settings in windows I get about 100fps
I have latest amdgpu pro drivers installed in ubuntu budgie 18.04.1 
I will try ferals gamemode today, hoping the performance will increase...
Anybody else here with this wrong recognized cpu problem?

edit: the native linux port of F1 2017 was much closer to the windows version, there I had round about 95% of the windows version, there was almost no difference...

**Comment by leillo1975** [comment]
Eveything works ok, except a framerate that's a bit low. In medium graphics about 50-55 fps in a GTX-1050Ti. Logitech G29 works with force feedback but with the effects inverted. There is not an option in game to invert FFB like in other simulation games. This night I will try it changing the Wheel compatibility mode to G27 or DFGT. You can use [Oversteer](https://github.com/berarma/oversteer) to change the modes (and for change range and test wheel)

**Comment by DanySK** [comment]
@leillo1975 would you report on your findings? Any luck with getting FF work as intended?
Is there any demo version around that could be used for testing?

**Comment by mozo78** [comment]
The game [runs](https://software.kaminata.net/linux-wine-games/f1-2018-dxvk-wine-linux/) fine and it requres only DXVK.

**Comment by DanySK** [comment]
@mozo78 have you tried with a steering wheel?

**Comment by mozo78** [comment]
Nope, I haven't one :)

**Comment by DanySK** [comment]
I'd be cautious, if wheels don't work, then the game does not run "fine" ;)

**Comment by kisak-valve** [comment]
F1 2018

Issue transferred from https://github.com/ValveSoftware/Proton/issues/4128.
@bthiebault posted on 2020-08-09T16:13:35:

# Compatibility Report
- Name of the game with compatibility issues: F1 2018
- Steam AppID of the game: 737800

## System Information
- GPU: NVIDIA GeForce RTX 2070 SUPER 
- Driver/LLVM version: 450.57
- Kernel version: 5.7.12-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/bthiebault/0b5fb7a7aaf6266287dd99f201d45cb2
- Proton version: 4.2-9

## I confirm:
- [X ] that I haven't found an existing compatibility report for this game.
- [X ] that I have checked whether there are updates for my system available.

[steam-737800.log](https://github.com/ValveSoftware/Proton/files/5047958/steam-737800.log)

## Symptoms 

During cut-scenes, the textures are very pixelated and models look ugly
![2020-08-09-180230_2560x1440_scrot](https://user-images.githubusercontent.com/50426075/89736631-f9c44700-da6a-11ea-8b9d-cb2a08c79238.png)

During the game, the graphics are good.

I have an XBox One (wired) controller. The vibrations are not correlated to the game events. I sometimes trigger when nothing is happening, but does not when I drive on the kerb.

Finally, changing the screen resolution in the settings freezes the game for quite some time, then succeeds.

## Reproduction

Launch the game

**Comment by Lepidos** [comment]
The game works almost flawlessly with **Logitech G29 Racing Wheel** if using a recent kernel then does support force feedback. But there is an issue with this game:
After finishing the 1st online race **Force Feedback** turns the wheel to the right, clockwise, and only stops making force after entering another race.

**Comment by bthiebault** [comment]
> F1 2018
> 
> Issue transferred from #4128.
> @bthiebault posted on 2020-08-09T16:13:35:
> 

> During the game, the graphics are good.
> 


Actually, I didn't notice at first, but some textures are bad as well during the game. For instance, the Renault writing on the car's protection ring is all blurry

**Comment by GenocideStomper** [comment]
Game doesn't start 90% of the time. I haven't figured out yet what helps, whether reboots help or not. I've tried 2 latest proton-GE versions and Proton experimental. Proton log hasn't told me anything obvious, but it contains a lot of personal data, so I would need to clean that up before attaching it.

Once in-game, it runs perfect.

Just wondering if anyone else has faced this strange issue.
